### ISSUE TITLE
wizardInstallSuggestedTest fails if ATH runs several times

### ISSUE THREAD

**Issue by NotMyFault** [jenkinsci/acceptance-test-harness#796]
Originally observed in https://github.com/jenkinsci/acceptance-test-harness/pull/794, `core.InstallWizardTest.wizardInstallSuggestedTest` fails if a job runs several times:
References: https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-794/3/testReport/junit/core/InstallWizardTest/java_11_jenkins_latest_split4___wizardInstallSuggestedTest/

**Comment by timja** [comment]
It's this:
bouncycastle API

Can be reproduced on the tip of master but not on 2.356.

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/21194782/175699372-da76ddd8-7ec7-4955-98ac-3d73f09bf26c.png">

**Comment by timja** [comment]
Bisected to https://github.com/jenkinsci/jenkins/pull/6585

**Comment by jglick** [comment]
(In the old days we would just release a plugin with an incremental dep. But this was error-prone—see https://github.com/jenkinsci/instance-identity-plugin/pull/21—and as of https://github.com/jenkinsci/plugin-pom/pull/416 + https://github.com/jenkins-infra/jenkins-maven-cd-action/issues/3 is not possible at all.)

**Comment by jglick** [comment]
> there is no release of `instance-identity` on the UC

See: https://github.com/jenkins-infra/update-center2/pull/594

**Comment by jglick** [comment]
Fixed now on Jenkins 2.357 AFAICT.

```
INFO	hudson.PluginManager#install: Starting installation of a batch of 19 plugins plus their dependencies
INFO	hudson.model.UpdateSite$Plugin#deploy: Adding dependent install of instance-identity for plugin cloudbees-folder
INFO	hudson.model.UpdateSite$Plugin#deploy: Adding dependent install of bouncycastle-api for plugin instance-identity
…
```
